R/lineage.R

```diff
@@ -395,10 +395,10 @@ IPG2Lineage <- function(accessions, ipg_file,
     {
         # browser()
         acc <- accessions[i]
-        acc_inds <- which(mergedTax$Protein == acc)
+        acc_inds <- which(.data$mergedTax$Protein == acc)
```

Comment:
This is not needed and makes me think that the IPG2Lineage() function defined in acc2lin.R may be more recent/relevant. `mergedTax` looks like it may have been pulled from the Global Environment at some point in the past -- it will not work as it exists in this function. 

Again, this is outside the scope of this issue, but something we will need to disambiguate (acc2lin.R vs lineage.R)/correct moving forward!